cluster/mesos/cluster.go

```diff
@@ -177,6 +177,15 @@ func (c *Cluster) UnregisterEventHandler(h cluster.EventHandler) {
 	c.eventHandlers.UnregisterEventHandler(h)
 }
 
+// StartContainer starts a container
+func (c *Cluster) StartContainer(container *cluster.Container) error {
+	// if the container was started less than a second ago, do not start it
+	if time.Now().Unix()-container.Created > 1 {
```

Comment:
yes, but if the container is already started, start does nothing so it's not an issue.

The issue is for cases like `docker run busybox ls`